### PR TITLE
Fix `github_organizations` is empty on merge

### PR DESCRIPTION
**Pull request by gcardozo123**

### CODE REVIEW

File: err_stash.py [hunk 1]
```diff
@@ -551,6 +551,10 @@ def merge(self, msg, branch_text, force=False):
         projects = self.config['STASH_PROJECTS']
         if not projects:
             return '`STASH_PROJECTS` not configured. Use `!plugin config Stash` to configure it.'
+        
+        github_organizations = self.config['GITHUB_ORGANIZATIONS']
```
Comment (nicoddemus) [inline err_stash.py]: Can we add a test please?

**Review by nicoddemus — CHANGES_REQUESTED** [review]

File: err_stash.py (hunk 1 above)
Comment (gcardozo123) [reply err_stash.py]: Sure, I'll do it!

Comment (nicoddemus) [reply err_stash.py]: Also plase review the README, I see the configuration for the bot is outdated.

Comment (gcardozo123) [reply err_stash.py]: Done

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**